crates/dbs-miniball/README.md

```diff
@@ -430,85 +398,81 @@ will be expanded as other devices are added. Block devices are in the works.
+Once all the prerequisites are met, the Miniball can be run either directly through `cargo`,
+passing on its specific [command line arguments](https://github.com/openanolis/dragonball-sandbox/tree/main/crates/dbs-miniball#cli-reference), 
+or after building it with `cargo build`.
 
-```wrap
+```shell
 cargo run --release --            \
     --memory size_mib=1024        \
     --kernel path=${KERNEL_PATH}  \
     --vcpu num=1
 ```
 
-```wrap
+```shell
 cargo build --release
-target/release/dbs-miniball      \
+target/release/dbs-miniball       \
     --memory size_mib=1024        \
     --kernel path=${KERNEL_PATH}  \
     --vcpu num=1
 ```
 
+Examples:
+```shell
+cargo run --release --            \
+    --memory size_mib=1024        \
+    --kernel path=/tmp/kernel/linux-5.4.81/vmlinux-hello-busybox  \
+    --vcpu num=1        \
+    --block path=/tmp/ubuntu-focal/rootfs.ext4
+```
+
+```shell
+cargo build --release
+target/release/dbs-miniball      \
+    --memory size_mib=1024        \
+    --kernel path=/tmp/kernel/linux-5.4.81/vmlinux-hello-busybox  \
+    --vcpu num=1        \
+    --block path=/tmp/ubuntu-focal/rootfs.ext4
+```
+
 ## Platform Supported
 
 ### Host OS & hypervisor
 
-`rust-vmm` is ultimately meant to be usable across a wide range of host OSes
-and hypervisors, with several of the crates already supporting Windows.
-Long-term, the Miniball should support the intersection of all its crates’
-supported platforms. Currently, this intersection resolves into Linux hosts
-and the KVM hypervisor. The first iteration of the Miniball supports
+Currently, this intersection resolves into `Linux hosts`
+and the `KVM hypervisor`. The first iteration of the Miniball supports
 only this configuration, returning errors when users attempt to run it on
 something else.
 
 ### CPU
 
-Long term, the Miniball will run on `x86_64` and `aarch64` platforms; to
-begin with, only Intel `x86_64` CPUs are supported, as there is no support
-for a PIO serial console on `aarch64`, and the virtio interfaces are at an
-early stage.
-
-### Rust features
-
-The Miniball will be compiled with all its crates’ features (at least,
-while they’re compatible with each other), and will be able to showcase usage
-with all of them in turn. Code-wise, there is only one `vmm` crate, which
-activates all its dependencies’ features through its TOML. The integration
-tests, documentation and examples will demonstrate the various features’
-functionalities. Examples of such features are: driving the event manager from
-the same thread or a new one, loading ELF and `bzImage` kernels.
-
-To exemplify this: the same Miniball binary can boot an ELF or a `bzImage`
-kernel, with no need to recompile it. Users control what it will boot, through
-the specified parameters (the user supplies the kernel image).
+Long term, the Miniball will run on `x86_64` and `aarch64` platforms. 
+Currently, only Intel `x86_64` CPUs are supported.
 
 ### Rust version
 
-Rust 1.60.0
+Rust 2018
```

Comment:
2018 is not the version number for Rust.  It's a Rust edition instead of version.

Reply:
fix